Sources/Console/Command/Command+Print.swift

```diff
@@ -15,9 +15,7 @@ extension Command {
     }
 
     public func printSignatureHelp() {
-        let maxWidth = getSignitureLength()
-        let leadingSpace = 2
-        let width = maxWidth + leadingSpace
+        let width = getSignitureLength(withPadding: 2)
```

Comment:
"Signature" has a typo here and where the function is declared below.

Reply:
Fixed.